src/reducers/arcReducer.ts

```diff
@@ -298,7 +298,7 @@ const arcReducer = (state = initialState, action: any) => {
 
   switch (action.type) {
     case ActionTypes.ARC_LOAD_CACHED_STATE_FULFILLED: {
-      return payload;
+      return payload
```

Comment:
?

Reply:
I changed this line a few times bc of the confusion about the location of the initialstate file (on staging or production...) and forgot to restore the `; `. 

Shoudl the linter have caught this?

Reply:
seems like it